packages/x-tree-view/src/internals/plugins/useTreeViewFocus/useTreeViewFocus.types.ts

```diff
@@ -7,12 +7,14 @@ import { UseTreeViewExpansionSignature } from '../useTreeViewExpansion';
 
 export interface UseTreeViewFocusInstance {
   isNodeFocused: (nodeId: string) => boolean;
-  focusNode: (event: React.SyntheticEvent, nodeId: string | null) => void;
-  focusDefaultNode: (event: React.SyntheticEvent) => void;
-  focusRoot: () => void;
+  isTreeViewFocused: () => boolean;
+  canNodeBeTabbed: (nodeId: string) => boolean;
+  focusNode: (event: React.SyntheticEvent, nodeId: string) => void;
+  focusDefaultNode: (event: React.SyntheticEvent | null) => void;
```

Comment:
I'm adding `null` on the event so that when you remove the focused node, it can switch to another node (was already the case before) AND call `onNodeFocus` for this new node (was not the case before).